EliteDangerous/DB/SystemClassDB.cs

```diff
@@ -199,11 +254,11 @@ public enum SystemAskType { AnyStars, PopulatedStars, UnPopulatedStars };
         }
 
 
-        public static List<long> GetEdsmIdsFromName(string name, SQLiteConnectionSystem cn = null , bool uselike = false)
+        public static List<long> GetEdsmIdsFromName(string name, SQLiteConnectionSystem cn = null, bool uselike = false)
         {
             List<long> ret = new List<long>();
 
-            if (name!=null && name.Length > 0)
+            if (name != null && name.Length > 0)
```

Comment:
`if (!string.IsNullOrEmpty(name))`

Reply:
For the record, I ran a stopwatch on a few billion iterations of this a few weeks ago, comparing cost of null + Len checks vs. the static method over a wide variety of inputs. `string.IsNullOrEmpty` was roughly 75% of the cost overall, unless the test cases were all null (at which case it was about 102%).